packages/extensions-sdk/src/cli/commands/validators/check-built-code.ts

```diff
@@ -0,0 +1,58 @@
+import { EXTENSION_PKG_KEY } from '@directus/extensions';
+import fse from 'fs-extra';
+import { type Ora } from 'ora';
+import path from 'path';
+import type { Report } from '../../types.js';
+
+const checkBuiltCode = {
+	name: 'built-code',
+	handler: async (spinner: Ora, reports: Array<Report>) => {
+		spinner.text = 'Check for built code';
+
+		let codePath = '/dist';
+		const packagePath = path.resolve('package.json');
+
+		if (await fse.pathExists(packagePath)) {
+			const packageFile = await fse.readJson(packagePath);
+
+			if (packageFile[EXTENSION_PKG_KEY]) {
+				const { path } = packageFile[EXTENSION_PKG_KEY];
```

Comment:
The `path` value could be an object that breaks subsequent checks.

```js
path: { app: 'dist/app.js', api: 'dist/api.js' },
```

Reply:
I think I've solved this on L36-L52 now